### ISSUE TITLE
State of "App Bar and Bounding Box"

### ISSUE THREAD

**Issue by brean** [Microsoft/MixedRealityToolkit-Unity#1497]
Hi,

In the [UX Readme](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/Assets/HoloToolkit-Examples/UX/README.md) there is a "comming soon" for the "App Bar and Bounding Box" example. In https://github.com/Microsoft/MRDesignLabs_Unity/issues/59 it is mentioned that it will be one of the next things that will get ported from MRDesignLabs.

What is the state for this and what is the problem with it? 
Do you need help?

**Comment by Railboy** [comment]
We've been working on multi-pointer support and other input prereqs for bounding box manipulation. Once #1486 is merged into the dev branch we'll create a new feature branch for the bounding box. In the meantime any input or suggestions you have for #1486 is always welcome.

**Comment by cre8ivepark** [comment]
@brean We opened a pull request for the Bounding Box normal mode with Two hand Gesture. 
#1727 
Adjust mode Bounding Box with handles will be added through a different pull request.

**Comment by davidkline-ms** [comment]
Shipped in the 2017.2.1.4 release!
